messages/en.json

```diff
@@ -46,16 +46,22 @@
         "clue": "Find the biggest drink cup for sale and take a photo next to it."
       },
       "-buCfuZ_Js": {
-        "answer": "Sub Pop, near Concourse C.",
+        "answer": {
+          "answer": "Sub Pop, near Concourse C."
+        },
         "clue": "You've got a postcard to mail. Where in the Central Terminal can you buy postage stamps?"
       },
       "1XWgzKECE6": {
-        "answer": "5 (to be verified)",
+        "answer": {
+          "answer": "5 (to be verified)"
+        },
         "clue": "Find the boba shop. How many different add-ins can you choose from?"
       },
       "4B8BuOoy-A": {
-        "answer": "26",
-        "answerDetails": "Gate D26 is located downstairs, across from Gate D5.",
+        "answer": {
+          "answer": "26",
+          "answerDetails": "Gate D26 is located downstairs, across from Gate D5."
```

Comment:
`answerDetails` is redundant, it should just be called `details` everywhere and accessed as `answer.details`.